extension/test/unit/payment-handler.spec.js

```diff
@@ -170,42 +168,37 @@ describe('payment-handler::execute', () => {
       expect(response.data.actions).to.deep.equal([])
     })
 
-    it('when amountPlanned is updated and multiple makePayment exists, ' +
```

Comment:
should not be those tests valid after your change, why updating it?

Reply:
They are not valid anymore as now we additionally check for both `makePaymentRequest` interface interaction and `makePaymentRequest` custom field.